### PR TITLE
#testfix: fixing system tests that were broken by recent feature addi…

### PR DESCRIPTION
**Pull request by thomasgauvin**
…tions

### CODE REVIEW

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (narrapangan) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java:1]: I think this is supposed to be small letter. Could you confirm @tarekait1996

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarSuggestionTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarSuggestionTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java (hunk 1 above)
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CampusSwitcherTest.java [hunk 3]
```diff
@@ -1,5 +1,4 @@
-package com.example.concordia_campus_guide.activities;
-
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/CampusSwitcherTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CurrentLocationButtonTest.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/CurrentLocationButtonTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/ExplorePOICardTest.java [hunk 5]
```diff
@@ -1,5 +1,4 @@
-package com.example.concordia_campus_guide.activities;
-
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/ExplorePOICardTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/ExplorePOIClickTest.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/ExplorePOIClickTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/FloorPickerAndMarkersHallTest.java [hunk 7]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/FloorPickerAndMarkersHallTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardHallTest.java [hunk 8]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardHallTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java [hunk 9]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/RoutesActivityOptionsTest.java [hunk 10]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/RoutesActivityOptionsTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/RoutesDirectionsHappyPathTest.java [hunk 11]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/RoutesDirectionsHappyPathTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST7SelectIndoorLocationTest.java [hunk 12]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST7SelectIndoorLocationTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST8SelectIndoorDestinationAndOriginTest.java [hunk 13]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST8SelectIndoorDestinationAndOriginTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST9SelectIndoorDestinationAndOriginOnDifferentCampuses.java [hunk 14]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/ST9SelectIndoorDestinationAndOriginOnDifferentCampuses.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/SearchActivityTest.java [hunk 15]
```diff
@@ -1,4 +1,4 @@
-package com.example.concordia_campus_guide.activities;
+package com.example.concordia_campus_guide.Activities;
```
Comment (viveanban) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/SearchActivityTest.java:1]:
```suggestion
package com.example.concordia_campus_guide.activities;
```

**Comment by viveanban** [comment]
Most of the test passed when I tried it out except for :
[calendarSuggestionTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463746/calendarSuggestionTest.log)
[routesActivityOptionsTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463747/routesActivityOptionsTest.log)
[calendarNotificationTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463748/calendarNotificationTest.log)

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java (hunk 1 above)
Comment (thomasgauvin) [reply app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java:1]: It is causing the tests to not be recognized so nothing is working @narrapangan @viveanban @tarekait1996

Comment (thomasgauvin) [reply app/src/androidTest/java/com/example/concordia_campus_guide/activities/CalendarNotificationTest.java]: Ok, for some reason I had a path with capital `Activities` but now its `activities`, so its working now

**Comment by thomasgauvin** [comment]
> Most of the test passed when I tried it out except for :
> [calendarSuggestionTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463746/calendarSuggestionTest.log)
> [routesActivityOptionsTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463747/routesActivityOptionsTest.log)
> [calendarNotificationTest.log](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/files/4463748/calendarNotificationTest.log)

Is it possible that you are not allowing permissions for the calendar/do not have the event in your calendar with a location `H-9, 937` in your calendar within the next hour?

**Comment by thomasgauvin** [comment]
Also, you must set your location to `Concordia University` on your emulator

**Comment by thomasgauvin** [comment]
All system tests passing

image

**Pull request merged**